code/game/response_team.dm

```diff
@@ -299,11 +299,9 @@ proc/trigger_armed_response_team(var/force = 0)
 	camera.c_tag = real_name
 
 	//Basic Uniform
-	equip_to_slot_or_del(new /obj/item/clothing/under/syndicate/tacticool(src), slot_w_uniform)
+	equip_to_slot_or_del(new /obj/item/clothing/under/ert(src), slot_w_uniform)
 	equip_to_slot_or_del(new /obj/item/device/flashlight(src), slot_l_store)
-	equip_to_slot_or_del(new /obj/item/weapon/clipboard(src), slot_r_store)
 	equip_to_slot_or_del(new /obj/item/weapon/gun/energy/gun(src), slot_belt)
-	equip_to_slot_or_del(new /obj/item/clothing/mask/gas/swat(src), slot_wear_mask)
```

Comment:
I'm replacing the one in their survival box with a gas mask that fits their aesthetics better. Plus, the ERT have a reputation of being heroes of space so it doesn't make sense to have them scare people with the same masks as nuke ops and deathsquids.
![mask](https://cloud.githubusercontent.com/assets/21122521/20896199/de4d52f8-bb1c-11e6-9a21-09628abff742.png)